.circleci/config.yml

```diff
@@ -687,7 +719,7 @@ workflows:
           requires:
             - eunit
             - static_analysis
-            - hodl
+            - hodl_blue
```

Comment:
As blue is deployed before green, this means that docker image (versioned and `latest`) pushed before green deployed. Ok. (Just confirming.)